sources/documents/2_RTB/Glossario/Lettere/d.tex

```diff
@@ -4,27 +4,56 @@ \section{D}
 \subsection*{Database}
 Un database è una raccolta organizzata di dati strutturati, che possono essere memorizzati e accessibili elettronicamente da un sistema informatico. I database sono progettati per consentire una gestione efficiente dei dati, facilitando operazioni di inserimento, aggiornamento, eliminazione e interrogazione. Utilizzano sistemi di gestione di database (DBMS) per garantire l'integrità, la sicurezza e la disponibilità dei dati.
 
+\vspace{2em}
+\subsection*{Database vettoriale}
+Un database vettoriale è progettato per indicizzare, archiviare e gestire grandi quantità di dati vettoriali altamente dimensionali. Alcuni esempi di database vettoriali sono FAISS (Facebook AI Similarity Search) e Pinecone, utilizzati in applicazioni di intelligenza artificiale e machine learning per effettuare ricerche di similarità e recupero di dati vettoriali, con funzionalità aggiuntive come operazioni \glossario{CRUD} e filtraggio dei metadati.
```

Comment:
Ma nei termini si devono riprendere parole che vengono già inserite nel glossario? Tipo CRUD in questo caso è inserita nel glossario, ma tipo FAISS no